tests/serverpod_test_server/lib/src/models/nullability.spy.yaml

```diff
@@ -24,6 +24,9 @@ fields:
   aUuid: UuidValue
   aNullableUuid: UuidValue?
 
+  aBigInt: BigInt
+  aNullableBigInt: BigInt?
```

Comment:
Suggestion: Commit the yaml files separately from the generated code.

Reply:
(It's a bit tricky to untangle this locally, as I first check everything working together and then do the 3 commits, but I also can't think of a better way right now (except maybe just 2 commits for YAML and "manual" code changes, and then the generated files – either way we'll have some "partial" commits in between that would not work by themselves).)